.gitignore

```diff
@@ -11,6 +11,9 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# IDEs
+.idea
+
```

Comment:
Please don't add this to the project. Otherwise we end up with each and every editor's config files in gitignore.
Here's what I do instead, use a global gitignore: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c